Software/Backend/WebApi/Controllers/UserController.cs

```diff
@@ -62,7 +62,7 @@ public ActionResult<GetUserResponse> GetUser(int id)
 
         return user;
     }
-
+    
```

Comment:
Ne mora se commitati ovako nešto.

Reply:
To je samo prazan red hahaha, ispod je API